menu_trans.vim.template

```diff
@@ -1,10 +1,13 @@
-" Menu Translations:    Simplified Chinese
-" Maintainer:           Shun Bai <baishunde@gmail.com>
-" Previous Maintainer:  Yuheng Xie <elephant@linux.net.cn>
```

Comment:
Should keep "Yuheng Xie <elephant@linux.net.cn>" in "previous maintainers", too.